src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.ts

```diff
@@ -386,6 +387,19 @@ export class RgwMultisiteDetailsComponent implements OnDestroy, OnInit {
     this.realmIds = [];
     this.zoneIds = [];
     this.getDisableMigrate();
+    this.rgwDaemonService.list().subscribe((data: any) => {
+      const zonegroupNames = data.map((item: { [x: string]: any }) => item['zonegroup_name']);
+      const zoneNames = data.map((item: { [x: string]: any }) => item['zone_name']);
+      if (
+        this.defaultRealmId != '' &&
+        this.defaultZonegroupId != '' &&
+        this.defaultZoneId != '' &&
+        zonegroupNames.includes('default') &&
+        zoneNames.includes('default')
```

Comment:
Can the name still be default and default after migrate?

Reply:
Updated the PR